io.github.niccokunzmann.python_dhcp_server.yml

```diff
@@ -0,0 +1,87 @@
+id: io.github.niccokunzmann.python_dhcp_server
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: python_dhcp_server
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
since it install tk are you sure this is compatible with wayland?

Reply:
It ran, I do not know. I can try removing this and see if it still works.

Reply:
When I removed these permissions, the app did not start at all, being unable to open a window.

Reply:
if you remove both you definitly have NO display. But do you run a wayland session? Because I can tell you tk does not support wayland. Which mean the only reason this would work is if you don't run a wayland sessions.